content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -327,10 +337,8 @@ The service accounts are:
   example](https://github.com/kubeflow/examples/tree/master/github_issue_summarization/pipelines).
```

Comment:
Lines 3, 8, 19, 27, 295, 298, 304, 317 and 325 (sorry, I couldn't comment on them directly):

- For consistency: `Google Cloud` ~~`Platform`~~/~~`(GCP)`~~, as you did in the other file.